src/griffe/docstrings/google.py

```diff
@@ -402,8 +437,13 @@ def _read_receives_section(docstring: Docstring, offset: int) -> tuple[Docstring
     return None, new_offset
 
 
-def _read_examples_section(docstring: Docstring, offset: int) -> tuple[DocstringSection | None, int]:  # noqa: WPS231
+def _read_examples_section(  # noqa: WPS231
+    docstring: Docstring,
+    offset: int,
+    **options: Any,
+) -> tuple[DocstringSection | None, int]:
     text, new_offset = _read_block(docstring, offset)
+    trim_doctest_flags = options.get("trim_doctest_flags", True)
```

Comment:
I guess it would be fine if you explicitly declare `trim_docstring_flags` in the signature while keeping `**options`. This would allow to both provide an annotation and a default without having to use `options.get` in the function's body.